.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,17 +6,21 @@ Closes <!--- Insert Issue Number(s) this PR addresses. Start by typing # will op
 
 ### All Submissions:
 
-* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
 * [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
 * [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.
 
+### Contribution Checklist
+
+* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
+* [ ] Have you associated this PR with a `type:` label?
+* [ ] Have you associated a milestone with this PR?
+* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?
```

Comment:
Can we leave this in the Author's checklist?

Reply:
This is all for releasenote automation that we have recently developed